infrastructure/keycloak/realm-export.json

```diff
@@ -839,7 +812,7 @@
       "enabled": true,
       "alwaysDisplayInConsole": false,
       "clientAuthenticatorType": "client-secret",
-      "secret": "ae18ccc9-06c2-4227-8c64-3d06d95511d6",
+      "secret": "**********",
```

Comment:
I think this key was hard-coded so that the regression testing would work. I don't think you'll want to be changing this .json file.

Reply:
I had to just make a change to the clients in order for the authentication to work. I didnt realize it would override anything else. I can just revert the secret I think